java/code/src/com/redhat/rhn/common/conf/UserDefaults.java

```diff
@@ -20,8 +20,6 @@
  */
 public class UserDefaults {
     private static UserDefaults instance = new UserDefaults();
-    public static final String MAX_PASSWORD_LENGTH = "max_passwd_len";
-    public static final String MIN_PASSWORD_LENGTH = "min_passwd_len";
```

Comment:
You should remove these configurations also from the "default config".
You find it in `python/spacewalk/rhn-conf/rhn_server.conf`